src/providers/consensus/typings.py

```diff
@@ -6,6 +6,28 @@
 from src.utils.dataclass import Nested, FromResponse
 
 
+@dataclass
+class BeaconSpecResponse(FromResponse):
+    DEPOSIT_CHAIN_ID: int
+    SLOTS_PER_EPOCH: int
+    SECONDS_PER_SLOT: int
+
+    def __post_init__(self):
```

Comment:
It can not be used as frozen. Is there any way to do it in __new__ and maybe do this in from_reponse?

Reply:
Not sure whether we should solve it in the current PR